clippy_lints/src/redundant_clone.rs

```diff
@@ -229,7 +229,7 @@ fn is_call_with_ref_arg<'tcx>(
     if_chain! {
         if let TerminatorKind::Call { func, args, destination, .. } = kind;
         if args.len() == 1;
-        if let mir::Operand::Move(mir::Place::Local(local)) = &args[0];
+        if let mir::Operand::Move(mir::Place::Base(PlaceBase(Local(local))) = &args[0];
```

Comment:
```suggestion
        if let mir::Operand::Move(mir::Place::Base(PlaceBase(Local(local)))) = &args[0];
```
It should fix the error.

Reply:
@mati865 doh but that's not the issue. `PlaceBase::Local` is the thing

Reply:
I admit I haven't checked it deeply, thanks for the explanation.